addons/calendar/tests/test_event_notifications.py

```diff
@@ -417,3 +418,45 @@ def search_event():
         with freeze_time('2023-11-15 19:00:00'):    # 14:00 the day before event
             self.assertEqual(len(search_event()), 0)
         event.unlink()
+
+    def test_calendar_recurring_event_delete_notification(self):
```

Comment:
In this test, could you please add some one line coments (#) through it explaining and dividing (with new lines):
- Test preparation
- Test execution
- Test expected results